File: 백준/Bronze/2953. 나는 요리사다/나는 요리사다.cc
#include<stdio.h>

int main() {
    int n, count=0, winner=0, max=0;
    for(int i=0; i < 5; i++){
        for(int x=0; x < 4; x++){
            scanf("%d", &n);
            count += n;
        }
        if(count > max){winner = i + 1;max = count;}
        count = 0;
    }
    printf("%d %d", winner, max);
}

File: 백준/Bronze/2953. 나는 요리사다/나는 요리사다.java
import java.util.Scanner;

public class Main {
    public static void main(String args[]) {
        Scanner sc = new Scanner(System.in);
        int n, count=0, winner=0, max=0;
        for (int i = 0; i < 5; i++){
            for(int x = 0; x < 4; x++){
                n = sc.nextInt();
                count += n;
            }
            if (count > max){
                winner = i + 1;
                max = count;
            }
            count = 0;
        }
        System.out.printf("%d %d", winner, max);
    }
}